how-to/how-to-use-rootless-kata-containers-with-podman.md

```diff
@@ -0,0 +1,275 @@
+        * [Missing registry file](#missing-registry-file)
+
+For an even more secure system, [Kata Containers](https://Katacontainers.io)
+can run workloads without a privileged user. Using
+[Podman](https://podman.io/) as the container engine, and
+[`slirp4netns`](https://github.com/rootless-containers/slirp4netns) for
+user-space networking.
+
+## Requirements
+
+- A Linux system, see
+  [supported distributions](https://github.com/kata-containers/documentation/blob/master/install/README.md#supported-distributions)
+  for an updated list.
+
+  - If using CentOS 7, `newuidmap` and `newgidmap` do not exist. Install them with:
+
+    ```bash
+    $ (git clone https://github.com/shadow-maint/shadow; cd shadow; ./autogen.sh --prefix=/usr --enable-man; make && sudo make -C src install)
+    $ sudo bash -c 'echo 10000 > /proc/sys/user/max_user_namespaces'
+    $ sudo bash -c "echo $(whoami):110000:65536 > /etc/subuid"
+    $ sudo bash -c "echo $(whoami):110000:65536 > /etc/subgid"
+    ```
+
+- Golang 1.12
+
+## Installation
+
+Refer to the following table for the __minimum__ required software versions
+and the installation instructions:
+
+| Component       | Version       | Install Instructions|
+| ----------------|:-------------:|---------------------|
+| Podman          | 1.6.2         | [see here](https://github.com/containers/libpod/blob/master/install.md)
+| `slirp4netns`   | 0.4.0         | [see here](https://github.com/rootless-containers/slirp4netns#quick-start)
+| Kata Containers | 1.10.0-alpha1 | [see here](https://github.com/kata-containers/documentation/blob/master/install/README.md)
+
+
+> **NOTE:**
+>
+> If installing Podman with a package manager, there is usually no need to
+> install `slirp4netns` separately.
+> You will need to install Kata Containers from source today for rootless support.
+> Rootless support for Kata should be available and functional in upcoming 1.9.2 and 1.10.0-alpha1 releases.
+
+## Configuration
+
+Now that you have installed Kata Containers and Podman, you need to configure
+them for rootless execution.
+
+### Disable SELinux
+
+> **Warning:**
+>
+> You should understand the security implications before disabling SELinux.
+
+If SELinux is installed and enabled, it needs to be disabled with the
+following command (Kata Containers
+[does not support SELinux](https://github.com/kata-containers/documentation/blob/master/Limitations.md#selinux-support)).
+
+> **Warning:**
+> The following command might differ depending on the distro you use:
+
+```bash
+$ [ -f /etc/selinux/config ] && sudo sed -i 's/^SELINUX=.*/SELINUX=disabled/g' /etc/selinux/config
+```
+
+### Add user to KVM group
+
+If running a KVM based hypervisor, add the user running the workload to the KVM group:
+
+```bash
+$ sudo usermod -a -G kvm $USER
+```
+> **NOTE:**
+>
+> `kvm` should be the group owning the device node `/dev/kvm` on most distros.
+> Make sure permissions on `/dev/kvm` are as shown below:
+> ```
```

Comment:
My permissions didn't exactly match this and was `crw-rw-rw-`   I suggest rewording to  "Make sure the minimum permissions on `/dev/kvm` are shown below"

Reply:
Fixed